app/templates/merchant/view_listings.html

```diff
@@ -9,22 +9,30 @@
         'success': get_flashed_messages(category_filter=['form-success'])
      } %}
 
-
+    {% if cart_listings is none %}
+        <a class="ui basic compact button" href="{{ url_for('admin.index') }}">
+            <i class="caret left icon"></i>
+            Back to dashboard
+        </a>
+    {% endif %}
+    
     <form class="ui form segment">
         <div class="twelve wide field">
             <label>Search names and descriptions</label>
             <input value="{{ main_search_term }}" type="text" name="main-search" placeholder="e.g. corned beef">
         </div>
 
-        <div class="inline field">
-            <div class="ui checkbox">
-                {% if favorite %}
-                    {% set checked = 'checked="checked"' %}
-                {% endif %}
-                <input type="checkbox" name="favorite" tabindex="0" {{ checked }}>
-                <label>Only show favorites</label>
+        {% if cart_listings %}
```

Comment:
It looks like now merchants can only search for favorites if they have an item in their cart. Let's separate the two views so we're not mixing merchant and admin views together.